packages/protocol/contracts/bridge/EtherVault.sol

```diff
@@ -104,6 +109,7 @@ contract EtherVault is EssentialContract {
     }
 
     /**
+     * TODO(dave): remove helper, can directly use `authorizedAddrs[addr]`.
```

Comment:
We can rename `authorizedAddrs` to `isAuthorized` and remove the function.

Reply:
i actually needed to keep isAuthorized(), because it may be useful externally as a getter (and was used in unit tests)